src/html/openstack_accounting/main.html

```diff
@@ -180,7 +180,7 @@ <h3> Openstack Quotas And Usage Monitoring </h3>
                 // UPDATE WHEN SCHEMA CHANGES!
                 "columnDefs": [
                   {
-                    "targets": [0, 14, 15, 16] ,
+                    "targets": [0, 14, 15, 16, 17] ,
```

Comment:
@Germano0 can you update the comment `// DEFAULT SHOW ONLY: ....` and add "contacts" column there too for future reference.

Reply:
@mrceyhun done